src/harness/testcases/WINNF_FT_S_GRA_testcase.py

```diff
@@ -327,37 +358,32 @@ def test_WINNF_FT_S_GRA_5(self):
                    } ]  },
      'grants': [ {
                'id': 'SAMPLE_ID_12345',
-               'operationParam': grant_0['operationParam'],
+               'operationParam': grant_a_SAS_TH['operationParam'],
                  'channelType': 'GAA',
                 'grantExpireTime': grant_expire,
                'terminated': False} ] }
     dump_file['recordData'].append(internal_data)
     device_a_id = dump_file['recordData'][0]['id']
 
-    expected_path = '/activity_dump_file/%s' % device_a_id
     # Setup the Server with response body and expected path from SAS Under Test
-    response = self._sas_server.setupServer({'responseBody': dump_file, 'expectedPath': expected_path})
+    self._sas_server.setupCbsdActivity(device_a['userId'], dump_file)
     activity_dump_file = {}
-    activity_dump_file['url'] = 'https://' + config['url'] + '/activity_dump_file/%s'% device_a_id
-    activity_dump_file['checksum'] = config['checksum']
-    activity_dump_file['size'] = config['size']
-    activity_dump_file['version'] = config['version']
+    activity_dump_file['url'] = 'https://' + url + '/cbsd_activity/%s'% device_a_id
```

Comment:
It needs to be clear somewhere that the URL should not include the "https://" -- or this code should be flexible enough to handle either case.